osu.Game/Screens/OnlinePlay/Multiplayer/CreateMultiplayerMatchButton.cs

```diff
@@ -10,14 +10,29 @@ namespace osu.Game.Screens.OnlinePlay.Multiplayer
 {
     public class CreateMultiplayerMatchButton : PurpleTriangleButton
     {
+        private IBindable<bool> isConnected;
+        private IBindable<bool> operationInProgress;
+
+        [Resolved]
+        private StatefulMultiplayerClient multiplayerClient { get; set; }
+
+        [Resolved]
+        private OngoingOperationTracker ongoingOperationTracker { get; set; }
+
         [BackgroundDependencyLoader]
-        private void load(StatefulMultiplayerClient multiplayerClient)
+        private void load()
         {
             Triangles.TriangleScale = 1.5f;
 
             Text = "Create room";
 
-            ((IBindable<bool>)Enabled).BindTo(multiplayerClient.IsConnected);
+            isConnected = multiplayerClient.IsConnected.GetBoundCopy();
+            isConnected.BindValueChanged(_ => updateState());
+
+            operationInProgress = ongoingOperationTracker.InProgress.GetBoundCopy();
```

Comment:
for safety we usually avoid doing binding logic in async load, can this be moved to `LoadComplete` without issue?

Reply:
Yep, this is just an oversight.